ab3d2_source/hires.s

```diff
@@ -6469,7 +6469,7 @@ groundfloor:
 				add.w	xwobzoff,d7
 
 				tst.b	Vid_FullScreen_b
-				beq.s	.shiftit
+				bra.s	.shiftit
```

Comment:
This branch us now unconditional, so the tst before it is no longer needed.

Reply:
In fact, I think all the code up to the .shiftit label is now unreachable so it could be completely removed, along with the branch here,

Reply:
commented out for now